keyboards/handwired/dactyl_manuform/5x6/keymaps/brownelvis/keymap.c

```diff
@@ -0,0 +1,64 @@
+/* A standard layout for the Dactyl Manuform 5x6 Keyboard */ 
+
+#include QMK_KEYBOARD_H
+
+
+enum layers {
+    _BASE,
```

Comment:
This is preventing the keymap from compiling.

```suggestion
    _QWERTY,
```